src/main/java/org/eclipse/yasson/JsonBindingProvider.java

```diff
@@ -22,6 +22,9 @@
  */
 public class JsonBindingProvider extends JsonbProvider {
 
+    public JsonBindingProvider() {
+    }
+
```

Comment:
As I wrote in commit [comment](https://github.com/api-from-the-ion/yasson/commit/a2966bb1d49f1040e5d294a675122747c211e73b) this is because of JPMS: these are the classes in the root package, which is exported. So I get warnings from the compiler, which suggests adding an explicit constructor. I think the public one is OK, because we should be able to create the instances of these classes outside the packages?